src/utils/metaMaskChainParameters.ts

```diff
@@ -118,6 +118,13 @@ export const METAMASK_CHAIN_PARAMETERS: {
     rpcUrls: ["https://testnet.aurora.dev"],
     blockExplorerUrls: ["https://testnet.aurorascan.dev"],
   },
+  1287: {
+    chainId: "0x507",
+    chainName: "Moonbase Alpha",
+    nativeCurrency: { name: "Dev", symbol: "DEV", decimals: 18 },
```

Comment:
I think the native currency should be GLMR

Reply:
@kev1n-peters
I can do that, that was my first choice but when I search for the explorer:
https://moonbase.moonscan.io/

They use `DEV` as their main currency.

But I'll do the change.

Reply:
Actually I see it as DEV in the explorer so 🤷‍♂️